.github/workflows/install_linux_from_binaries.yaml

```diff
@@ -6,14 +6,20 @@ on:
   schedule:
     - cron: '0 0 * * *'
 
-
 jobs:
   ubuntu-install-from-binaries:
     runs-on: ubuntu-latest
```

Comment:
Should be fix the `ubuntu-version` here ?

Reply:
Addressed in 33d1ae6